open-apparel-registry/open-apparel-registry
# Display claim info (OSH)

## Overview

Ports the facility claim information section from the old sidebar to the new sidebar, removes old sidebar components, and adds a claimed badge to claim items.

This is a duplicate of [#1901](https://github.com/open-apparel-registry/open-apparel-registry/pull/1901) for OSH. The OSH sidebar contains sector information, but otherwise changes should be the same. 


Connects #1901 

## Demo

<img width="358" alt="Screen Shot 2022-06-14 at 3 08 11 PM" src="https://user-images.githubusercontent.com/21046714/173669805-b2ce0cd5-342a-4ed2-b342-cc6e9692cce8.png">

## Testing Instructions

* Run `./scripts/server` and view a claimed facility. Note that claims information should be displayed, as well as sector information.

## Checklist
